### PR TITLE
Fix preg line return

### PR DESCRIPTION
**Pull request by gallart**

### CODE REVIEW

File: data/webshells.txt [hunk 1]
```diff
@@ -5,4 +5,5 @@ eval\(base64_decode\(
 "base" \. "64_decode"
 ; for\(\$i=0; \$i < strlen\(
 \?php \${
-=\$GLOBALS;\${
\ No newline at end of file
+=\$GLOBALS;\${
+\$GLOBALS\[\$GLOBALS\['
```
Comment (soullivaneuh) [inline data/webshells.txt]: I don't think this is part of the preg_line fix.

What is it? This should be on a separated commit.

File: scan.php [hunk 2]
```diff
@@ -175,38 +175,38 @@ function patch_file($file, $content)
         foreach ($files as $file) {
             if (is_dir($file) === true) { // Not in use, was used to check directory traversal was working properly
             } else { // If is file
-                        if (preg_match("/\.php$/", $file)) { // Currently only selects PHP scripts for scanning
```
Comment (soullivaneuh) [inline scan.php:179]: This tab correction must be on a separated commit.

**Review by soullivaneuh — CHANGES_REQUESTED** [review]

File: scan.php (hunk 2 above)
Comment (gallart) [reply scan.php:179]: Don't know how to fix that.

Comment (soullivaneuh) [reply scan.php:179]: As usual: Undo your commit. Add your tab modification with `git add --patch`, commit, then commit again for the rest. ;-)

**Pull request closed without merging**